games-action/minetest/metadata.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<herd>games</herd>
```

Comment:
> Can you point out please where the issue with sci-libs/libspatialindex lies?

https://github.com/minetest/minetest/blob/master/src/CMakeLists.txt#L183

It just silently disables the feature if it is not found, hence we get automagic linking.